roles/logging/templates/etc/rsyslog.d/60-remote-syslog.conf

```diff
@@ -0,0 +1,47 @@
+$WorkDirectory {{ logging.syslog_forwarding.work_directory }}
+
+{% if logging.syslog_forwarding.action_resume is defined %}
+{% if logging.syslog_forwarding.action_resume.retry_count is defined %}
```

Comment:
I wasn't sure what to do with with_dict (maybe use it to find a regexp and replace it with line?) . But I changed it so it goes through a list of variables instead of a bunch of `if` statements. Please lmk what you think. Thanks.